packages/react/src/Navbar.js

```diff
@@ -8,13 +8,6 @@ import {
   Container,
 } from "react-bootstrap";
 
-let Link = "a";
-try {
-  Link = require("react-router-dom").Link;
```

Comment:
This block of code is causing StoryBook to fail during dev. 
Link also was used as a value as ={ Link } of web component instead of <Link > </Link>